src/tests/ftest/telemetry/telemetry_pool_space_metrics.yaml

```
@@ -0,0 +1,73 @@
+hosts:
+  test_servers: 2
+  test_clients: 1
+timeout: 120
+server_config:
+  name: daos_server
+  engines_per_host: 2
+  engines:
+    0:
+      targets: 4
+      pinned_numa_node: 0
+      nr_xs_helpers: 1
+      fabric_iface: ib0
+      fabric_iface_port: 31317
+      log_file: daos_engine_0.log
+      storage:
+        0:
+          class: dcpm
+          scm_list: ["/dev/pmem0"]
+          scm_mount: /mnt/daos0
+        1:
+          class: nvme
+          bdev_list: ["aaaa:aa:aa.a"]
```

Newer way - ftest will automatically figure this out
```suggestion
      storage: auto
```

---

- [ ] Use auto device management